spec/double_write_cache_stores/client_spec.rb

```diff
@@ -364,21 +364,6 @@ def get_or_read(store, key)
           it_behaves_like "cache store example", one_cache_store
         end
       end
-
-      context "ActiveSupport :dalli_store in Dalli" do
```

Comment:
Why remove these specs?

Reply:
This test was removed in https://github.com/mixigroup/double_write_cache_stores/commit/5b0bdaaa57764e9f3abba94e2aa576093bee98b1, probably because dalli_store was removed from dalli 3.0.
cf. https://stackoverflow.com/a/76655389

Reply:
Ok, that sounds good.
Could you update `Support backend cache store` in README.md ?

Reply:
Fixed README 🙏  b7255a5